### PR TITLE
Add ASCII export option

### PR DESCRIPTION
**Pull request by emiliopedrollo**
add option to print on _stdout_ without any additional symbols and already with line breaks where they are supposed to be.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,18 +35,24 @@ async function main () {
...
+
     const result = await findPrime(options.number, options.sophie)
-    log.info(options.export === 'json' ? JSON.stringify(result, undefined, 2) : result.prime)
+
+    if (options.export === 'json')
+        log.info(JSON.stringify(result,undefined,2))
+    else if (options.export === 'prime')
+        log.info(result.prime)
+    else
+        result.prime.match(new RegExp(`.{1,${options.width * 2}}`,'g')).forEach((line) => console.log(line))
```
Comment (TotalTechGeek) [inline index.js]: Could we instead do a:
```
import { splitEvery } from 'ramda'  // at the top
// and in place of that, do: 
log.info(splitEvery(options.width * 2, result.prime).join('\n'))

File: readme.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Options:
   -n, --number <number>  The number to be transformed into a prime.
   -i, --image <image>    Use an image to find primes.
   -q, --quiet            Hides some of the debug information to make it easier to get output from this program.
-  -x, --export <mode>    The output format (choices: "json", "prime", default: "json")
+  -x, --export <mode>    The output format (choices: "json", "prime", "ascii" default: "json")
```
Comment (TotalTechGeek) [inline readme.md]: Missing a comma :)

**Review by TotalTechGeek — CHANGES_REQUESTED** [review]
Great MR, I like the functionality, might request a few minor changes,

File: index.js (hunk 1 above)
Comment (emiliopedrollo) [reply index.js]: The log.info may print unnecessary symbols on the output (for coloring and other stuff) that's the reason for choosing console.log.

File: readme.md (hunk 2 above)
Comment (emiliopedrollo) [reply readme.md]: done

**Pull request merged**